app/views/blocks/addressQuestion.html

```diff
@@ -41,8 +41,9 @@
       ng-required="block.required"
       class="form-control"
       ng-disabled="editBlock"
+      ng-if="answer.value.country === 'US'"
     >
-      <option value="">State</option>
+      <option value="" disabled selected>State</option>
```

Comment:
You should generate these from your JS array so they aren't hardcoded multiple places...